### PR TITLE
Code to send PunchHistory and Comments To PCS5

### PR DESCRIPTION
**Pull request by TordJoranger**
resolves [equinor/cc-toolbox-tasks#60](https://github.com/equinor/cc-toolbox-tasks/issues/60)

### CODE REVIEW

File: Core/Models/Person.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 namespace Core.Models;
 
-public class Person : IPersonEventV1
+public struct Person : IPersonEventV1
```
Comment (jsoi-eq) [inline Core/Models/Person.cs:5]: Curious: Why?

**Comment by jsoi-eq** [comment]
Missing AB#123456 in description

File: Core/Models/Person.cs (hunk 1 above)
Comment (TordJoranger) [reply Core/Models/Person.cs:5]: it removed some squiggly lines, and I don't need it to have a reference in memory (I hope)

**Comment by TordJoranger** [comment]
> Missing AB#123456 in description

fixed, although not to azure devops anymore.

File: FamFeederFunction/Functions/FamFeeder/TopicHttpTrigger.cs [hunk 2]
```diff
@@ -74,7 +74,11 @@ public static List<string> SplitList(string input)
 
     public static bool HasValidTopic(string topic)
     {
-          return  TopicHelper.GetAllTopicsAsEnumerable().Contains(topic, StringComparer.InvariantCultureIgnoreCase);
+        return topic switch
+        {
+            "PunchItemHistory" or "PunchItemComments" => true,
```
Comment (jsoi-eq) [inline FamFeederFunction/Functions/FamFeeder/TopicHttpTrigger.cs:79]: Are we going to get more special cases in other parts of the code / other repos by omitting to add these to PcsTopicConstants, or is this the only one...?

Comment (TordJoranger) [reply FamFeederFunction/Functions/FamFeeder/TopicHttpTrigger.cs:79]: I've chosen to not add these to PcsBusService, as they are not used by other repos than this. 
The plan is also to remove this code after Oracle no longer is master for this data.

Couple of more are coming. Attachments atleast

File: Infrastructure/Repositories/EventRepository.cs [hunk 3]
```diff
@@ -69,6 +69,17 @@ public async Task<IEnumerable<string>> GetPunchItemsForCompletion(string plant)
         const string extraClause = " and p.isVoided = 'N'";
         return await Query<PunchListItem>(PunchListItemQuery.GetQuery(null, plant, extraClause));
     }
+
+    public async Task<IEnumerable<string>> GetPunchItemHistory(string plant)
```
Comment (jsoi-eq) [inline Infrastructure/Repositories/EventRepository.cs:73]: No expression body to make it consistent with the others?

**Review by jsoi-eq — APPROVED** [review]
Approving pending your decision on the minor issues I've commented on.

**Pull request merged**